kratos/tests/cpp_tests/spatial_containers/test_geometrical_objects_bins.cpp

```diff
@@ -79,6 +79,35 @@ namespace Testing {
         KRATOS_CHECK_NEAR(bounding_box.GetMaxPoint()[2], cube_z, tolerance);
     }
 
+    
+    /** Checks modified constructor 
+    */
+    KRATOS_TEST_CASE_IN_SUITE(GeometricalObjectsBinsConstructor, KratosFastSuite) {
+        constexpr double tolerance = 1e-12;
+
+        Model current_model;
+
+        const double cube_x = 0.6;
+        const double cube_y = 0.9;
+        const double cube_z = 0.3;
+
+        // Generate the cube skin
+        ModelPart& skin_part = CreateCubeSkinModelPart(current_model, cube_x, cube_y, cube_z);
+
+        array_1d<double,3> cell_size{0.1,0.1,0.1};
```

Comment:
I would check with different sizes in each direction (may hide copy paste errors)

Reply:
true thanks!